template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Extensions/AzureAd/AzureAdServiceCollectionExtensions.cs

```diff
@@ -40,7 +40,7 @@ public PostConfigureAzureOptions(IOptions<AzureAdOptions> azureOptions)
 
             public void PostConfigure(string name, JwtBearerOptions options)
             {
-                options.Audience = _azureOptions.ClientId;
+                options.Audience = _azureOptions.Audience;
```

Comment:
I don't think this change would be correct, the idea is that the JwtBearer audience should be the AzureAd.ClientId.  @danroth27 has the most context, can you confirm?